guide/daily-spending-wallet/requesting.md

```diff
@@ -34,439 +34,454 @@ Illustration source
+{% include /tip/close.html %}
 
 <div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/opening-channel.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/opening-channel@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddress.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddress@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddress@2x.png"
    layout = "shadow"
-   caption = "Using Lightning services, a channel can be opened on-demand if there is no inbound liquidity when a payment is received."
-   alt-text = "Screen showing channel being opened."
+   caption = "Payment addresses should are managed in the settings."
+   alt-text = "Screen showing the users settings with a QR icon in the top left that opens a payment address."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/channel-opened.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/channel-opened@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddressQR.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddressQR@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddressQR@2x.png"
    layout = "shadow"
-   caption = "Indicate to users that, if they have no inbound liquidity, they should get some before requesting a payment."
-   alt-text = "Screen showing that inbound liquidity is required."
+   caption = "This wallet can quickly open and share a payment address by clicking the QR code in the top left of the settings page."
+   alt-text = "Screen showing QR code of a payment address."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/funding/no-inbound.png"
-   retina = "/assets/images/guide/daily-spending-wallet/funding/no-inbound@2x.png"
+   image = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddresses.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddresses@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/PaymentAddresses@2x.png"
    layout = "shadow"
-   caption = "Guide users as to how to obtain inbound liquidity so they can receive payments."
-   alt-text = "Screen showing options to receive inbound liquidity."
+   caption = "Allow users to manage payment addresses separately  if your app supports multiple."
```

Comment:
Let's discuss separately if we want to allow for managing multiple payment address, and whether it should be possible to add ones that this wallet cannot generate (lightning address, LNURL-Pay).

Reply:
I say we just remove this for now and explore it independently.